straxen/plugins/high_energy_processing.py

```diff
@@ -0,0 +1,121 @@
+import strax
+import straxen
+import numpy as np
+
+export, __all__ = strax.exporter()
+
+# These are also needed in peaklets, since hitfinding is repeated
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=False,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'See straxen.hit_min_amplitude for options.'
+    )])
+
+
+@export
+@strax.takes_config(
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(straxen.PulseProcessing):
+    __version__ = '0.2.5'
```

Comment:
You could also start with version 0.0.0 right?

Reply:
Also for other plugins

Reply:
I'm not a paying attention to details kind of guy